examples/hello_world/test_driver/smoke_web_engine_test.dart

```diff
@@ -35,12 +37,25 @@ void main() {
 
       await Future<void>.delayed(const Duration(seconds: 2));
 
+      // A flutter web app may be rendered directly on the body of the page, or
+      // inside the shadow root of the flt-glass-pane (after Flutter 2.4). To
+      // make this test backwards compatible, we first need to locate the correct
+      // root for the app.
+      //
+      // It's either the shadowRoot within flt-glass-pane, or [driver.webDriver].
+      final SearchContext? shadow = await driver.webDriver.execute(
+        'return document.querySelector("flt-glass-pane")?.shadowRoot;',
+        <dynamic>[],
+      ) as SearchContext?;
+
+      final SearchContext appRoot = shadow ?? driver.webDriver;
```

Comment:
I'm going to simplify this to one line, and that way we can kick the build again :)

Reply:
Done.